src/helpers.js

```diff
@@ -40,4 +40,36 @@ export const cloneObject = (object) => {
         results[key] = data;
     });
     return results;
+};
+
+export const hasDiffrentKeysLength = (srcObject, compareObject) => {
+    return Object.keys(srcObject).length !== Object.keys(compareObject).length;
+};
+
+export const isNotEqualArray = (srcArray, compareArray) => {
```

Comment:
Why not name those variables properly? You have compareArray, so why no sourceArray instead of srcArray?